webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -390,9 +390,10 @@ public function viewAction(
                     $runResult['hostname'] = $firstJudgingRun->getJudgeTask()->getJudgehost()->getHostname();
                     $runResult['judgehostid'] = $firstJudgingRun->getJudgeTask()->getJudgehost()->getJudgehostid();
                 }
-                $runResult['is_output_run_truncated'] = $outputDisplayLimit >= 0 && preg_match(
+                $runResult['is_output_run_truncated'] = preg_match(
```

Comment:
We indicate whether the database field `output_run` (in the table `judging_run_output`) is truncated. It is orthogonal to the display limitation.

Now,
- if the display is not limited, we store the whole database field `output_run` into `$runResult['output_run']`
- if the display is limited, we truncate the first N bytes of the database field `output_run` into `$runResult['output_run']` and **additionally** we store the last X bytes of the database field `output_run` into `$runResult['output_run_last_bytes']` (otherwise it is not set)
 
Then we use this info to decide how to label the buttons.

Reply:
Ok, I follow now, also because I now understand how `$runResult['is_output_run_truncated']` is used in the template to render buttons for downloading the DB stored output run or fetch if from the judgehost via a judgetask.

I do think that this name could be improved because it refers to whether the DB data is truncated, but it's easy to interpret as that `$runResult['output_run']` is truncated, which is different because it can also be truncated by the display limit.

Reply:
Renamed it to `is_output_run_truncated_in_db` - is that the change you had in mind?